js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js

```diff
@@ -1431,4 +1431,25 @@
       }
     }
   };
+
+  window.searchHelper = {
+    skipEvent: function (event) {
+      if (
+        event &&
+        event.originalEvent &&
+        ((event.originalEvent.key &&
+          (event.originalEvent.key === "Control" ||
+            event.originalEvent.key === "Alt" ||
+            event.originalEvent.key === "Shift" ||
+            event.originalEvent.key === "Meta")) ||
+          event.originalEvent.ctrlKey ||
+          event.originalEvent.altKey ||
+          event.originalEvent.metaKey)
+      ) {
+        return true;
+      }
+      return false;
```

Comment:
Optional suggestion:

```suggestion
      return (
        event &&
        event.originalEvent &&
        ((event.originalEvent.key &&
          (event.originalEvent.key === "Control" ||
            event.originalEvent.key === "Alt" ||
            event.originalEvent.key === "Shift" ||
            event.originalEvent.key === "Meta")) ||
          event.originalEvent.ctrlKey ||
          event.originalEvent.altKey ||
          event.originalEvent.metaKey)
      ) 
```

Reply:
let's be explicit, we can keep it as it is